packages/editor-ui/src/App.vue

```diff
@@ -278,12 +281,12 @@ export default defineComponent({
 
 .header {
 	grid-area: header;
-	z-index: 999;
+	z-index: 99;
```

Comment:
Decrease here to allow the v1 banner to display on top of header and sidebar, but without having to increase the banner's `z-index` so much that it would even display on top of the backdrop used for modals.